src/Agent.Worker/AgentLogPlugin.cs

```diff
@@ -36,6 +36,7 @@ public sealed class AgentLogPlugin : AgentService, IAgentLogPlugin
         private readonly Dictionary<string, PluginInfo> _logPlugins = new Dictionary<string, PluginInfo>()
         {
             {"TestResultLogPlugin",  new PluginInfo("Agent.Plugins.Log.TestResultParser.Plugin.TestResultLogPlugin, Agent.Plugins", "Test Result Parser plugin")},
+            {"TestFilePublisherPlugin",  new PluginInfo("Agent.Plugins.Log.TestFilePublisher.Plugin.TestFilePublisherLogPlugin, Agent.Plugins", "Test File Publisher plugin")}
```

Comment:
do we need 2 plugins for test result? any chance we can get down to one?

Reply:
@TingluoHuang have thought about it. 
* Both plugins require independent turn on/turn off mechanism.
* Their working model/design is different. 
* We are planning to optimize the Auto Test file publisher by reading some of the log file which is different design than parsers.